server/main.go

```diff
@@ -131,3 +123,9 @@ func HealthHandler(w http.ResponseWriter, r *http.Request) {
 	fmt.Fprint(w, "health check ok")
 	log.Info("health check ok")
 }
+
+func handleError(e error) {
+	if e != nil {
+		log.Error(e)
```

Comment:
log.Error() only sends to logs? Shouldn't these go to rollbar? Also, most of the places we're calling this, an error likely means the following operations won't work. Should we return and abandon further processing when we get these errors?

Reply:
My understanding is that rollrus adds a hook to the logger that sends to rollbar as well, but I could be wrong: https://github.com/heroku/rollrus/blob/master/examples_test.go

Reply:
@joshwlewis ^

Reply:
But to your point - I think log.Panic would make more sense here over log.Error.